gears/src/baseapp/abci.rs

```diff
@@ -220,7 +220,7 @@ impl<DB: Database, PSK: ParamsSubspaceKey, H: ABCIHandler, AI: ApplicationInfo>
 
         ResponseCommit {
             data: hash.to_vec().into(),
-            retain_height: (height - 1), // TODO: check if this is correct - why -1?
+            retain_height: 0, // this is the height above which tendermint will retain all blocks // TODO: make this configurable as in Cosmos
```

Comment:
I have a feeling that we have more TODO that open issues. Maybe its worth to open one for this "issue"?

Reply:
You're right. I'll open an issue.